.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ executors:
   integration-tests:
     docker:
       - image: cimg/node:18.14-browsers
-      - image: cimg/redis:6.2
+      - image: cimg/redis:7.0
```

Comment:
we still use redis 6 in production
https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-incentives-prod/resources/elasticache.tf#L13

Reply:
Oh, makes sense to keep using the same version of the Redis as well then, I'll revert 👍

Reply:
Reverted: 2c332aceb2294f57b4a1ae57542f82c5d284fb59